README.md

```diff
@@ -95,6 +95,18 @@ Then import and use __`rustls`__ in the code as usual.
 
 (Unlike the original __`rustls`__, no features are enabled by default in this fork.)
 
+Note that this fork provides a crate-level [`Arc` type alias][`Arc`] to help use the correct `Arc` type according to the build configuration:
+
+- alias to [`portable_atomic_util::Arc`](https://docs.rs/portable-atomic-util/latest/portable_atomic_util/struct.Arc.html),
+if `RUSTFLAGS` is set with `--cfg unstable_portable_atomic_arc` during Cargo build
+- otherwise alias to [`alloc::sync::Arc`] / [`std::sync::Arc`](https://doc.rust-lang.org/nightly/std/sync/struct.Arc.html)
```

Comment:
aargh this reference to `alloc::sync::Arc` does not render as a reference in README.md